stage #2.5/module #4 REST Basics - Spring Web/Spring Rest/spring-rest-controller-content.md

```diff
@@ -0,0 +1,86 @@
+## Rest Controller
+
+Let's consider how to build REST controllers using Spring Web framework.
+Spring REST controllers are annotated with **@RestController**. The annotation includes the **@RestController** and **@ResponseBody** annotations, and as a result, simplifies the controller implementation:
+```Java
+@RestController
+@RequestMapping(value = "/books", consumes = {"application/JSON"}, produces = {"application/JSON", "application/XML"})
+public class BookController {
+    @Autowired
+    private BookService bookService;
+
+    @Autowired
+    private Convertor<BookDTO, Book> convertor;
+
+    @GetMapping
+    ResponseEntity<List<BookDTO>> getAllBooks(@RequestParam(defaultValue = "10", required = false) int limit, @RequestParam(defaultValue = "5", required = false) int offset) {
```

Comment:
As for me, we know that we work with Books based on the name of the controller.
getAllBooks, getBookByIdcan be replaced like getAll, getById and so on.